1436790/1438462: Part 13: 19 files with MSDN link fixes. (#10351)
dotnet/docs: docs/framework/data/adonet/index.md

 Describes features and behaviors that are specific to the .NET Framework Data Provider for Oracle.  
  
## Related Sections  

>  [Security in Native and .NET Framework Code](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/1787tk12(v=vs.100))  

Author: I removed the security link per your comment from part 11 (PR #10330) earlier: https://github.com/dotnet/docs/pull/10330#discussion_r254457351

 [WCF Data Services 4.5](../../../../docs/framework/data/wcf/index.md)  
 Describes how to use [!INCLUDE[ssAstoria](../../../../includes/ssastoria-md.md)] to deploy data services on the Web or an intranet that implement the Open Data Protocol (OData).  
  
 [Core Development Technologies](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.0/csxbhtye(v=vs.85))  
 Provides links to information about standard development tasks in the .NET Framework.  
  
 [Advanced Development Technologies](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.0/a493kdy6(v=vs.85)
)  
 Provides links to information about sophisticated development tasks and techniques in the .NET Framework.  
  
 [Samples and Quickstarts](https://docs.microsoft.com/previous-versions/ms184422(v=vs.100))  

>  Provides complete listings of all samples and QuickStart samples associated with the .NET Framework.  

Reviewer: ```suggestion
 Provides a list of .NET samples and tutorials.
```
